<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IO</class>
 <widget class="QWidget" name="IO">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>1330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="io_box">
     <property name="minimumSize">
      <size>
       <width>700</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Ebrima</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>FT2232_CH</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="send_box">
        <property name="title">
         <string>Send</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QTextEdit" name="send_in"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="receive_box">
        <property name="title">
         <string>Receive</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTextBrowser" name="receive_out"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" alignment="Qt::AlignLeft">
       <widget class="QGroupBox" name="functions">
        <property name="title">
         <string>Functions</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="1">
          <widget class="QCheckBox" name="multi_send">
           <property name="text">
            <string>Multi send Delay(ms):</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="send_enter">
           <property name="text">
            <string>Send Enter</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="delay">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="1" alignment="Qt::AlignVCenter">
       <widget class="QPushButton" name="send_botton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
